invenio_accounts/config.py

```diff
@@ -187,6 +190,25 @@
 SECURITY_CHANGE_URL = '/account/settings/password/'
 """URL endpoint for password change."""
 
+SECURITY_MSG_LOCAL_LOGIN_DISABLED = ("Local login is disabled.", "error")
```

Comment:
Just wondering: why having all these messages configurable? Do you see the need for it?